routes/articles.js

```diff
@@ -0,0 +1,101 @@
+const express = require('express');
+const router = express.Router();
+const knex = require('../database');
+
+router.route('/')
+  .get(function (req, res) {
+    knex.select('urltitle', 'title', 'author', 'body', 'urltitle').from('articles').orderBy('updated_at', 'desc')
```

Comment:
Normally you wouldn't put all the methods on one line, it's easier to read if each method was on its own line.